selfdrive/car/hyundai/values.py

```diff
@@ -215,7 +215,7 @@ class Buttons:
   "use_elect_gears": [CAR.KIA_OPTIMA_H, CAR.IONIQ_EV_LTD, CAR.KONA_EV, CAR.IONIQ],
 
   # these cars use the FCA11 message for the AEB and FCW signals, all others use SCC12
-  "use_fca": [CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ, CAR.KONA, CAR.KONA_EV, CAR.KIA_FORTE, CAR.PALISADE],
+  "use_fca": [CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ_EV_LTD, CAR.KONA, CAR.KONA_EV, CAR.KIA_FORTE, CAR.PALISADE],
```

Comment:
This change should be made in another PR.

Reply:
reverted. possibly not an issue